### PR TITLE
Implement init fault detection

### PR DESCRIPTION
**Pull request by Nolle10**

### CODE REVIEW

File: src/fault_detection/wait-for-it.sh
Comment (Nolle10) [inline src/fault_detection/wait-for-it.sh:1]: Haven't read this shit... but it works allright!

**Review by Nicolaj2903 — APPROVED** [review]

File: src/fault_detection/heartbeat.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import (
+	"log"
+	"time"
+
+	"github.com/streadway/amqp"
+)
+
+// Service defines the structure for a monitored service
+type Service struct {
```
Comment (Crazi12345) [inline src/fault_detection/heartbeat.go:11]: bruh

**Review by Crazi12345 — APPROVED** [review]
# THIS IS AWFUL
## Just kidding
### Not Really
#### But Good Enough

File: src/fault_detection/heartbeat.go (hunk 1 above)
Comment (Nolle10) [reply src/fault_detection/heartbeat.go:11]: [https://tenor.com/view/spongebob-too-easy-basic-gif-17920371738562039657](url)

**Comment by Nolle10** [comment]
> # THIS IS AWFUL
> ## Just kidding
> ### Not Really
image

**Pull request merged**